src/applications/personalization/profile360/reducers/vet360.js

```diff
@@ -10,10 +10,19 @@ import {
   VET360_TRANSACTION_UPDATE_FAILED
 } from '../actions';
 
+import {
+  isFailedTransaction,
+  isSuccessfulTransaction
+} from '../util/transactions';
+
 const initialState = {
   transactions: [],
   fieldTransactionMap: {},
-  transactionsAwaitingUpdate: []
+  transactionsAwaitingUpdate: [],
+  metadata: {
+    mostRecentSuccessfulTransactionId: '',
+    mostRecentErroredTransactionId: '',
+  }
```

Comment:
I'm not sure if `metadata` is the best term here, but I was trying to use a term broad enough that we could store other information as we need it.